src/de.hpi.swa.trufflesqueak.test/src/de/hpi/swa/trufflesqueak/test/runCuisTests.st

```diff
@@ -9,7 +9,6 @@ nonTerminatingTestCases := OrderedCollection new.
 {
     #JpegTest -> #(#test16bpp61 #test16bpp63 #test16bppLE #test16bppLE61 #test16bppLE62 #test16bppLE63 #test32bpp #test32bpp63 #test32bppLE63 #testBA32bpp #testBA32bpp63 "all previous tests pass on TruffleSqueak in JVM mode, the rest is not yet supported" #test16bpp #test16bpp62 #testBA16bpp #testBA16bpp61 #testBA16bpp62 #testBA16bpp63 #testBA16bppLE #testBA16bppLE61 #testBA16bppLE62 #testBA16bppLE63 #testGray #testGray61 #testGray62 #testGray63 #testGrayBA #testGrayBA61 #testGrayBA62 #testGrayBA63).
     #ProcessTest -> #(#testResumeWithEnsureAfterBCR #testTerminateInEnsure).
-    #SemaphoreTest -> #("unexpected null reads" #testSemaInCriticalWaitInEnsure #testSemaInCriticalWaitInEnsure2).
```

Comment:
Looks like this still causes an assertion error that we haven't fixed yet: https://github.com/hpi-swa/trufflesqueak/actions/runs/15034462552/job/42263054508#step:14:817

Maybe revert the line?